docker-compose.yml

```diff
@@ -37,9 +37,9 @@ services:
     image: nginx
     volumes:
       - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
-      - /etc/ssl/certs/inferno:/etc/ssl/certs/inferno:ro
+      # - /etc/ssl/certs/inferno:/etc/ssl/certs/inferno:ro
```

Comment:
It looks like changing this line is needed for testing locally (based on the comment below it), but we ultimately don't want to pull this into master right?

Reply:
Yes, that's accurate. I meant to pull out all my testing changes, but missed this one. Reverting now.

Reply:
yup no worries 👍 .  Though I do still need to manually make this change to test locally with `docker build -t infernocommunity/inferno-reference-server:latest .` right?